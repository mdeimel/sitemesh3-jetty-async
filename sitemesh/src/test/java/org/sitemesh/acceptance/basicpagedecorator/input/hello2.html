<html>
  <head>
    <title>Hello 2</title>
    <style type="text/css">
      /* page specific css */
      .note { font-weight: bold; }
    </style>
  </head>
  <body>

    <p class="note">This document also has some addtional &lt;head&gt; tag contents.</p>

  </body>
</html>