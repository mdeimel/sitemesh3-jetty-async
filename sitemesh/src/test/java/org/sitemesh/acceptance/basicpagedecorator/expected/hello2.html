<!-- Common header -->
<html>
  <head>
    <title>Hello 2</title>
    <style type="text/css">
      /* common css */
      body { padding: 20px; }
    </style>
    <style type="text/css">
      /* page specific css */
      .note { font-weight: bold; }
    </style>
  </head>
  <body>

    <h1 style="text-align:center">Hello 2</h1>

    <!-- body -->
    <p class="note">This document also has some addtional &lt;head&gt; tag contents.</p>
    <!-- end of body -->

  </body>
</html>
<!-- Common footer -->